### PR TITLE
Prototype for Profile page, needs functional and visual adjustments

### PR DESCRIPTION
**Pull request by everamal**

### CODE REVIEW

File: src/app/components/login-profile/login-profile.component.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    this.model.name = "Calvin Klein";
+    this.model.image = "https://material.angular.io/assets/img/examples/shiba2.jpg";
+    this.model.email = "calvin@klein.com"
+    this.model.lastWeekAverage = "56";
+    this.model.currentWeekAverage = "32";
+    this.model.arrival = 8;
+  }
+
+  login() {
+    console.log(this.model.pin);
```
Comment (alfavenegas) [inline src/app/components/login-profile/login-profile.component.ts]: Do not push console logs!

File: src/app/Models/profile.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export class Profile {
+    pin: string;
```
Comment (JAntonioP10) [inline src/app/Models/profile.ts]: pin must be number in profile model.

File: src/app/Services/profile.service.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProfileService {
+
+  constructor(private httpClient: HttpClient) { }
+
+  getProfile(pin: number) {
+    return this.httpClient.post(`${environment.apiUrl}users/methodname`, { pin: pin });
```
Comment (JAntonioP10) [inline src/app/Services/profile.service.ts]: methodname?, see the routes on api project.

File: src/app/Services/profile.service.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  }
+
+  saveProfileChanges(profileChanges: Profile){
+    return this.httpClient.post(`${environment.apiUrl}users/methodname`, profileChanges);
```
Comment (JAntonioP10) [inline src/app/Services/profile.service.ts]: methodname?, see the routes on api project.

File: src/app/components/login-profile/login-profile.component.ts (hunk 1 above)
Comment (everamal) [reply src/app/components/login-profile/login-profile.component.ts]: Done

**Comment by JAntonioP10** [comment]
Routes.
Update user information: users/update-data
Get user information: users/user-profile

File: src/app/Services/profile.service.ts (hunk 3 above)
Comment (everamal) [reply src/app/Services/profile.service.ts]: can you provide me the name?

File: src/app/Services/profile.service.ts (hunk 4 above)
Comment (everamal) [reply src/app/Services/profile.service.ts]: can you provide me the name?

Comment (everamal) [reply src/app/Services/profile.service.ts]: Done

File: src/app/Services/profile.service.ts (hunk 3 above)
Comment (everamal) [reply src/app/Services/profile.service.ts]: Done

File: src/app/Models/profile.ts (hunk 2 above)
Comment (everamal) [reply src/app/Models/profile.ts]: Done

**Pull request merged**